package.json

```diff
@@ -6,6 +6,7 @@
     "type": "git",
     "url": "https://github.com/solana-labs/example-token"
   },
+  "testnetDefaultChannel": "beta",
```

Comment:
not stable?

Reply:
Dependabot is bumping dependencies on master including web3.js so we probably want to keep master tracking beta.

Reply:
@mvines  Any objection?